README.md

```diff
@@ -69,10 +69,6 @@ Generate coverage for a single module, i.e.:
 ### Building a binary release gzipped tar ball ###
     ./gradlew clean releaseTarGz
 
-The above command will fail if you haven't set up the signing key. To bypass signing the artifact, you can run:
-
-    ./gradlew clean releaseTarGz -x signArchives
```

Comment:
@ijuma the proper way to skip signing is now to run `./gradlew clean releaseTarGz -DskipSigning`, is that right? Is there a reason we removed this instruction from the README instead of updating it?

Reply:
Signing is skipped by default. Are you seeing something different? There is a table that describes `skipSigning`:
* https://github.com/apache/kafka#common-build-options

Reply:
Yes, we're suddenly seeing our soak test applications which deploys from trunk fail with

```
./gradlew clean install releaseTarGz
> Configure project :
Building project 'core' with Scala version 2.13.5
Building project 'streams-scala' with Scala version 2.13.5
> Task :connect:signMavenJavaPublication FAILED
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':connect:signMavenJavaPublication'.
> Cannot perform signing task ':connect:signMavenJavaPublication' because it has no configured signatory
```

I personally haven't been able to reproduce this, but both @wcarlson5 and @lct45 have reported running into this issue

Reply:
Note that your command has `install` while the README line I deleted did not (and was wrong). It is still weird that you're seeing this for snapshot builds. I'll work with you all offline to try and figure it out.

Reply:
We figured it out, the build was using a non snapshot version. Passing `-PskipSigning=true` fixed it. Submitted https://github.com/apache/kafka/pull/10307 to make it easier to debug these issues in the future.